bigbluebutton-html5/imports/api/2.0/users/server/methods/validateAuthToken.js

```diff
@@ -2,7 +2,7 @@ import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 import RedisPubSub from '/imports/startup/server/redis2x';
 import Logger from '/imports/startup/server/logger';
-import Users from '/imports/api/2.0/users';
+import Users from './../../';
```

Comment:
Hmm, kinda hard to know which one is being imported here. 2.0 or 1.1. IMO absolute paths are better in cases like this one.